test/controllers/address/ClosePostalAddressControllerSpec.scala

```diff
@@ -232,6 +233,42 @@ class ClosePostalAddressControllerSpec extends AddressBaseSpec {
       verify(controller.editAddressLockRepository, times(0)).insert(meq(nino.withoutSuffix), meq(PostalAddrType))
     }
 
+    "render the thank you page upon successful submission of closing the correspondence address and only a lock on the primary address" in new LocalSetup {
+      override def currentRequest[A]: Request[A] = FakeRequest("POST", "/test").asInstanceOf[Request[A]]
+      override def getEditedAddressIndicators: List[AddressJourneyTTLModel] =
+        List(AddressJourneyTTLModel("SomeNino", EditPrimaryAddress(BSONDateTime(0))))
+      val result = controller.confirmSubmit(FakeRequest())
+
+      status(result) shouldBe OK
```

Comment:
How do we know if this is redirecting to the thank you page?

Reply:
expanded the test